### PR TITLE
feat: add product tour experiment

### PR DESCRIPTION
**Pull request by alangsto**
## [MST-2143](https://2u-internal.atlassian.net/browse/MST-2143)

Add product tour as part of optimizely experiment.

Screenshot 2023-09-20 at 2 49 49 PM

Product tour is displayed until a user has clicked "Check it out".

### CODE REVIEW

File: src/utils/optimizelyExperiment.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import optimizelyInstance from '../data/optimizely';
+
+const PRODUCT_TOUR_EXP_KEY = 'la_product_tour';
+const PRODUCT_TOUR_EXP_VARIATION = 'learning_assistant_product_tour';
+
+const activateProductTourExperiment = (userId) => {
+  const variant = optimizelyInstance.activate(
```
Comment (alangsto) [inline src/utils/optimizelyExperiment.js:7]: variant is always returning `null` right now, even though it should be returning the variant.

File: src/utils/optimizelyExperiment.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    PRODUCT_TOUR_EXP_KEY,
+    userId,
+  );
+  // const variation = optimizelyInstance.getVariation(PRODUCT_TOUR_EXP_KEY, userId);
```
Comment (alangsto) [inline src/utils/optimizelyExperiment.js:11]: This returns the correct variation (that I'd also expect to be returned by the call to `activate`).

File: package.json [hunk 3]
```diff
@@ -45,9 +45,11 @@
     "@fortawesome/free-regular-svg-icons": "5.15.4",
     "@fortawesome/free-solid-svg-icons": "5.15.4",
     "@fortawesome/react-fontawesome": "0.2.0",
+    "@optimizely/react-sdk": "^2.9.2",
     "core-js": "3.31.1",
     "prop-types": "15.8.1",
-    "react-markdown": "^8.0.5"
+    "react-markdown": "^8.0.5",
+    "uuid": "9.0.0"
```
Comment (alangsto) [inline package.json:52]: UUID has to be moved to `dependencies` due to https://github.com/optimizely/javascript-sdk/issues/791

**Review by MichaelRoytman — APPROVED** [review]
Looks great!

**Pull request merged**